src/app/zap_cluster_list.json

```diff
@@ -171,7 +173,7 @@
             "concentration-measurement-server"
         ],
         "CHANNEL_CLUSTER": ["channel-server"],
-        "CHIME_CLUSTER": ["chime-server"],
+        "CHIME_CLUSTER": [],
```

Comment:
This does not seem related?  And looks wrong...

Reply:
Reverted.  No longer accidentally backing out this change.